services/chainstorage/src/test/java/tech/pegasys/teku/services/chainstorage/EphemeryDatabaseResetTest.java

```diff
@@ -55,30 +59,44 @@ void setUp() throws IOException {
     MockitoAnnotations.openMocks(this);
     ephemeryDatabaseReset = spy(new EphemeryDatabaseReset());
     beaconDataDir = createTempDirectory("beaconDataDir");
+    dbDataDir = beaconDataDir.resolve("db");
+    networkFile = new File(beaconDataDir.toFile(), "network.yml");
     final Path validatorDataDir = createTempDirectory("validatorDataDir");
     resolvedSlashProtectionDir = validatorDataDir.resolve("slashprotection");
+
     when(serviceConfig.getDataDirLayout()).thenReturn(dataDirLayout);
     when(dataDirLayout.getBeaconDataDirectory()).thenReturn(beaconDataDir);
+    when(dataDirLayout.getBeaconDataDirectory().resolve("db")).thenReturn(dbDataDir);
+    when(dataDirLayout.getBeaconDataDirectory().resolve("network.yml"))
+        .thenReturn(networkFile.toPath());
     when(dataDirLayout.getValidatorDataDirectory()).thenReturn(validatorDataDir);
     when(dataDirLayout.getValidatorDataDirectory().resolve("slashprotection"))
         .thenReturn(resolvedSlashProtectionDir);
   }
 
   @Test
-  void shouldResetDirectoriesAndCreateDatabase() throws IOException {
+  void shouldResetSpecificDirectoriesAndCreateDatabase() throws IOException {
+    final Path kvStoreDir = beaconDataDir.resolve("kvstore");
+    Files.createDirectory(kvStoreDir);
+    final Path dbVersion = beaconDataDir.resolve("db.version");
+    Files.createFile(dbVersion);
+
     when(dbFactory.createDatabase()).thenReturn(database);
+
     final Database result = ephemeryDatabaseReset.resetDatabaseAndCreate(serviceConfig, dbFactory);
-    verify(ephemeryDatabaseReset).deleteDirectoryRecursively(beaconDataDir);
+    verify(ephemeryDatabaseReset).deleteDirectoryRecursively(dbDataDir);
+    verify(ephemeryDatabaseReset).deleteFile(networkFile.toPath());
```

Comment:
this is pretty close now i think, we probably just also want to `verifyNoMoreInteractions` after what we expect... in basically every test if we're defining explicitly what we want.